woocommerce/woocommerce-gutenberg-products-block
# Remove redundant front-end Checkout block test case

We have one redundant [front-end test case file](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/frontend/checkout.test.js) for the Checkout block.

The `should display an empty cart message when cart is empty` test is already present in the [cart-checkout/checkout.test.js test file](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/shopper/cart-checkout/checkout.test.js).

In this PR we have: 

- Removed `should display an empty cart message when cart is empty` test case as we already had one in [cart-checkout/checkout.js.test](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/shopper/cart-checkout/checkout.test.js)
- Moved `allows customer to choose available payment methods` test to [cart-checkout/checkout.js.test](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/shopper/cart-checkout/checkout.test.js)
- Removed [frontend/checkout.test.js file](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/frontend/checkout.test.js) as all test cases are already covered in [cart-checkout/checkout.js.test](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/tests/e2e/specs/shopper/cart-checkout/checkout.test.js)


### Testing

#### Automated Tests

 
### Manual Testing

How to test the changes in this Pull Request:

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e-dev -- tests/e2e/specs/shopper/cart-checkout/checkout.test.js` to run only this test case.
5. Make sure all tests pass.
